pytensor/configdefaults.py

```diff
@@ -349,7 +349,7 @@ def add_compile_configvars():
 
     # Try to find the full compiler path from the name
     if param != "":
-        newp = find_executable(param)
+        newp = shutil.which(param)
```

Comment:
This might be slightly different between windows v linux v unix.

Also does `shutil` reference any aliases? Is that necessary?

Reply:
(not blocking just curious)

Reply:
No idea tbh, I only followed the deprecation recommendation